docs/cloud/automation/terraform.md

```diff
@@ -75,16 +75,31 @@ You can can use the [cloud console][cloud console tokens] or the [EvenStoreDB Cl
 
 The command to obtain a token from the [EvenStoreDB Cloud CLI][esc cli github releases] is:
 
- `esc access tokens create --email <email>`
+ ``` bash
+ esc access tokens create --email <email>
+ 
+ Password:
+ Token created for audience https://api.eventstore.cloud
+ OeTjTf***********************************0IWP
+  ```
+
+In the cloud console:
+::: card
+![token in cloud console](./images/token_console.png)
+:::
 
 ### Obtaining the `organization_id`
 
 You can use the [cloud console][cloud console organizations] or the [EvenStoreDB Cloud CLI][esc cli github releases] to find your organization id.
 
 The command to obtain the organization from the [EvenStoreDB Cloud CLI][esc cli github releases] is:
 
- `esc resources organizations list`
+``` bash
+ esc resources organizations list
+ Organization { id: OrgId("bt****************80"), name: "Event Store Ltd"
+```
 
+In the cloud console 
```

Comment:
**CONCERN** Is it copy/paste or the image is missing?

Reply:
added a screenshot & direct link to the organisation page